facefusion/face_analyser.py

```diff
@@ -400,3 +411,19 @@ def filter_by_gender(faces : List[Face], gender : FaceAnalyserGender) -> List[Fa
 		if categorize_gender(face.gender) == gender:
 			filter_faces.append(face)
 	return filter_faces
+
+
+def detect_landmark_68(frame : VisionFrame, bbox : Bbox) -> numpy.ndarray[Any, Any]:
```

Comment:
def detect_face_landmark_68(frame : VisionFrame, bbox : Bbox) -> FaceLandmark68

Reply:
It will only return one face landmark depends on face bbox provided.